ibm/service/power/resource_ibm_pi_volume_onboarding.go

```diff
@@ -39,11 +39,11 @@ func ResourceIBMPIVolumeOnboarding() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 
-			helpers.PICloudInstanceId: {
-				Type:        schema.TypeString,
-				Required:    true,
-				ForceNew:    true,
+			Arg_CloudInstanceID: {
 				Description: "Cloud Instance ID - This is the service_instance_id.",
+				ForceNew:    true,
+				Required:    true,
+				Type:        schema.TypeString,
 			},
 
 			piOnboardingVolumes: {
```

Comment:
All of these should be moved to the constant file and rename with `Arg_` prefix
eg:
`Arg_OnboardingVolumes = "pi_onboarding_volumes"`
```
const (
	piOnboardingVolumes   = "pi_onboarding_volumes"
	piAuxiliaryVolumes    = "pi_auxiliary_volumes"
	piAuxiliaryVolumeName = "pi_auxiliary_volume_name"
	piSourceCRN           = "pi_source_crn"
	piDisplayName         = "pi_display_name"
	piDescription         = "pi_description"
)
```